.travis.yml

```diff
@@ -65,18 +67,20 @@ jobs:
         - python run_pylint.py --disable=fixme --fail-under=8.5 vermouth 
         - python run_pylint.py --disable=fixme --fail-under=9.5 bin/martinize2
       after_success: skip
-#   - stage: deploy
-#     python: "3.7"
-#     addons: skip
-#     env:
-#     script: skip
-#     after_success: skip
-#     deploy:
-#         user: ...
-#         password:
-#             # travis encrypt --add deploy.password
-#             secure: ...
-#         provider: pypi
-#         on:
-#             tags: true
-
+    - stage: deploy
+      python: "3.7"
+      addons: skip
+      env:
+      script: skip
+      after_success: skip
```

Comment:
Do you need the explicitly skip the steps?

Reply:
Otherwise it'll run the tests again. I can try without this at a later point.

Reply:
I did not think of that. Thanks.